scripts/extract-translations.sh

```diff
@@ -1,3 +1,6 @@
+# https://github.com/lingui/js-lingui/issues/952#issuecomment-765449330
+export NODE_ENV=development
```

Comment:
why are we setting node env to dev here? also any reason we use this .sh file pattern for `extract` but not `compile`?

Reply:
It is, just being used directly inline:
```
    "i18n:compile": "NODE_ENV=development lingui compile",
    ```

Reply:
i meant the pattern of adding a separate .sh file

nbd, just asking for sake of consistency